### PR TITLE
ADD: goto button from search view

### PR DESCRIPTION
**Pull request by iTitus**
image

### CODE REVIEW

File: src/main/resources/static/img/download.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg class="bi bi-download" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+  <path fill-rule="evenodd" d="M.5 8a.5.5 0 01.5.5V12a1 1 0 001 1h12a1 1 0 001-1V8.5a.5.5 0 011 0V12a2 2 0 01-2 2H2a2 2 0 01-2-2V8.5A.5.5 0 01.5 8z" clip-rule="evenodd"/>
+  <path fill-rule="evenodd" d="M5 7.5a.5.5 0 01.707 0L8 9.793 10.293 7.5a.5.5 0 11.707.707l-2.646 2.647a.5.5 0 01-.708 0L5 8.207A.5.5 0 015 7.5z" clip-rule="evenodd"/>
+  <path fill-rule="evenodd" d="M8 1a.5.5 0 01.5.5v8a.5.5 0 01-1 0v-8A.5.5 0 018 1z" clip-rule="evenodd"/>
+</svg>
```
Comment (realemx) [inline src/main/resources/static/img/download.svg]: Missing new line

File: src/main/resources/static/img/list.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg class="bi bi-list" width="1em" height="1em" viewBox="0 0 16 16" fill="white" xmlns="http://www.w3.org/2000/svg">
+  <path fill-rule="evenodd" d="M2.5 11.5A.5.5 0 013 11h10a.5.5 0 010 1H3a.5.5 0 01-.5-.5zm0-4A.5.5 0 013 7h10a.5.5 0 010 1H3a.5.5 0 01-.5-.5zm0-4A.5.5 0 013 3h10a.5.5 0 010 1H3a.5.5 0 01-.5-.5z" clip-rule="evenodd"/>
+</svg>
```
Comment (realemx) [inline src/main/resources/static/img/list.svg]: Missing new line

File: src/main/resources/templates/groups.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
                 integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
                 crossorigin="anonymous"></script>
-        <script language="javascript" type="text/javascript" src="../static/script.js" th:src="@{/script.js}"></script>
+        <script src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```
Comment (realemx) [inline src/main/resources/templates/groups.html]:
```suggestion
        <script type="text/javascript" src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```
Bare minimum. Not all browsers autodetect JS.

File: src/main/resources/templates/mops_error.html [hunk 4]
```diff
@@ -17,7 +17,7 @@
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
                 integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
                 crossorigin="anonymous"></script>
-        <script language="javascript" type="text/javascript" src="../static/script.js" th:src="@{/script.js}"></script>
+        <script src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```
Comment (realemx) [inline src/main/resources/templates/mops_error.html]:
```suggestion
        <script type="text/javascript" src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```

File: src/main/resources/templates/overview.html [hunk 5]
```diff
@@ -17,7 +17,7 @@
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
                 integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
                 crossorigin="anonymous"></script>
-        <script language="javascript" type="text/javascript" src="../static/script.js" th:src="@{/script.js}"></script>
+        <script src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```
Comment (realemx) [inline src/main/resources/templates/overview.html]:
```suggestion
        <script type="text/javascript" src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```

File: src/main/resources/templates/overview.html [hunk 6]
```diff
@@ -102,10 +102,9 @@
         <!-- Button to the directory above -->
         <div th:if="${directory.parentId} != null">
             <div class="material1-button-above-wrapper">
-                <form action="#" th:action="@{/material1/dir/{dirId}(dirId = ${directory.parentId})}"
-                      th:method="get">
-                    <input class="btn btn-outline-primary" type="submit" value="Ebene hoch">
-                </form>
+                <a class="btn btn-outline-primary" th:href="@{/material1/dir/{dirId}(dirId = ${directory.parentId})}">
+                    Ebene hoch
```
Comment (realemx) [inline src/main/resources/templates/overview.html]: Maybe we could use this one here https://icons.getbootstrap.com/icons/arrow-90deg-up/

File: src/main/resources/templates/groups.html (hunk 3 above)
Comment (iTitus) [reply src/main/resources/templates/groups.html]: According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script the type and language attributes should not be used

File: src/main/resources/static/img/list.svg (hunk 2 above)
Comment (iTitus) [reply src/main/resources/static/img/list.svg]: I copied them from bootstrap icons, but can add

File: src/main/resources/templates/groups.html (hunk 3 above)
Comment (realemx) [reply src/main/resources/templates/groups.html]: Stupid HTML5, I'm too old for this.

**Pull request merged**